assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -76,7 +76,15 @@ const SearchFunnelWidget = ( {
 
 	const viewOnly = useViewOnly();
 
+	const isAnalyticsAvailable = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleAvailable( 'analytics' )
+	);
+
 	const canViewSharedAnalytics = useSelect( ( select ) => {
+		if ( ! isAnalyticsAvailable ) {
+			return false;
+		}
+
```

Comment:
This shouldn't be necessary because `canViewSharedModule` already checks for `module === null` internally. If anything, we should probably update that selector to use `isModuleAvailable` internally.

Reply:
I think we still do need this check to avoid always returning `true` when in view-only mode...